test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4036,7 +4036,7 @@ def log_lines(self, logtype, id=None, n=50, tail=True, starttime=None,
                     n = n - len(day_lines)
                     if n <= 0:
                         break
-        except (BaseException, IOError, PtlLogMatchError):
+        except (Exception, IOError, PtlLogMatchError):
```

Comment:
Exception seems kind of broad?  The only exceptions that should be thrown are from du.run_cmd(), du.cat(), and PtlLogMatchError.  What gets thrown from the two du functions?

Reply:
As of now du.run_cmd and du.cat doesn't throw any particular exception so it could be anything, so we have to Exception only.

Reply:
I still think Exception is a little broad, but at least it is better than a blanket except like it was.  I'm fine with it.